GNUmakefile

```diff
@@ -244,3 +250,466 @@ deploy-images:
 	@IMAGE=$(ENGINE_IMG) ./build/push
 	@IMAGE=$(PAIR_IMG) ./build/push
 	@IMAGE=$(DS_IMG) ./build/push
+	@echo ""
+	@echo "Pushing developer image...."
+	@docker push $(DEV_IMAGE)
```

Comment:
We don't need to push `DEV_IMAGE`, right? It is used during build only. So we can generate the DEV_IMAGE first and proceed with the build.

Reply:
Agreed. Removed.